lib/vm/README.md

```diff
@@ -19,6 +19,6 @@ docker run --rm -v $(pwd):/code \
   --mount type=volume,source=$(basename $(pwd))_cache,target=/code/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
   confio/cosmwasm-opt:0.6.2 ./contracts/hackatom
-cp contracts/hackatom/target/wasm32-unknown-unknown/release/hackatom.wasm \
+cp contracts/hackatom/contract.wasm \
```

Comment:
Oh this explains why I checked-in the 1.7 MB contract … Thanks

Reply:
Yeah, that shocked me... then I realized there was a small one lying around